Documentation/README_InteractiveElement.md

```diff
@@ -0,0 +1,91 @@
+# Interactive Element 
+
+An Interactive Element is the next generation of MRTK's Interactable. An Interactive Element is an object that receives input and fires events depending on the type of input.  The type of input supported is defined in the form of states. 
```

Comment:
>The type of input supported is defined in the form of states.
That is confusing to me -- what does this mean? Can you give an example?

Reply:
The type of input supported is referring to the types of input MRTK currently supports.   

Here are the current types of input supported in MRTK:
 
- Hand input (near and far interaction)
   - The `Touch` state is only supported via Hand input
- Controller input (VR, only far interaction)
  - We can define a state called `VR Controller Select` that is only set to on if the input source is a VR controller. 
- Touch screen input (2D only far interaction)
   - We can define a `Touch Screen Select` state
- Voice input (Far interaction)
  - The `Speech Keyword` state will be added

Each type of input will have its associated states. 

A better phrasing might be:  Each form of input MRTK supports (Hand, Controller, Touch Screen, Voice, etc.) has its own associated states.

For example if we have an object and we want it to only respond to a Select if the input source is a VR controller, then we could create a `VR Controller Select` state.  Adding this state would focus on the events for a specific type of input, which in this case is the VR Controller input.